lektor_webpack_support.py

```diff
@@ -24,9 +26,13 @@ def run_webpack(self, watch=False):
         return portable_popen(args, cwd=webpack_root)
 
     def npm_install(self):
-        reporter.report_generic('Running npm install')
-        webpack_root = os.path.join(self.env.root_path, 'webpack')
-        portable_popen(['npm', 'install'], cwd=webpack_root).wait()
+        if self.npm_lock.acquire(False):
```

Comment:
Can you specify `blocking=False` here? I had to look up the documentation for `threading.Lock` to understand what this `False` was for.